apps/financial-aid/web-veita/src/components/LogoMunicipality/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import cn from 'classnames'
+
+import LogoSvg from './LogoSvg'
+import { useQuery } from '@apollo/client'
+import { GetMunicipalityIdQuery } from '@island.is/financial-aid-web/veita/graphql'
+import { useRouter } from 'next/router'
+import { LoadingDots } from '@island.is/island-ui/core'
+
+interface MunicipalityData {
+  municipality: {
+    id: string
+    settings: {
+      homePage: string
+    }
+  }
+}
+
+interface LogoProps {
+  className?: string
+}
+
+const LogoMunicipalitycipality = ({ className }: LogoProps) => {
+  const router = useRouter()
+
+  const municipaliyId = router.query.sveitarfelag
+    ? (router.query.sveitarfelag as string)
+    : 'hfj'
```

Comment:
In the near future won't we get this from the staff but not from the url?

Reply:
Yess I added TODO there!